mllib/src/test/scala/org/apache/spark/ml/feature/MinMaxScalerSuite.scala

```diff
@@ -57,8 +58,7 @@ class MinMaxScalerSuite extends SparkFunSuite with MLlibTestSparkContext with De
 
   test("MinMaxScaler arguments max must be larger than min") {
     withClue("arguments max must be larger than min") {
-      val dummyDF = spark.createDataFrame(Seq(
-        (1, Vectors.dense(1.0, 2.0)))).toDF("id", "feature")
+      val dummyDF = Seq((1, Vectors.dense(1.0, 2.0))).toDF("id", "feature")
```

Comment:
It's just a column name, but for consistency...`features` (not `feature`) (unless there's a reason for this)